src/components.tsx

```diff
@@ -30,3 +35,68 @@ export class ErrorBoundary extends React.Component {
     return this.props.children;
   }
 }
+
+export function JupyterWidget({
+  widget,
+  view,
+}: {
+  widget: WidgetModel;
+  view: DOMWidgetView;
+}) {
+  const [element, setElement] = React.useState<HTMLElement | null>(null);
+
+  React.useLayoutEffect(() => {
+    let currentView: any = null;
+    let destroyed = false;
+
+    if (element != null) {
+      view.create_child_view(widget).then((newView) => {
+        currentView = newView;
+
+        // since create view is async, the vue component might be destroyed before the view is created
```

Comment:
vue? :)

Reply:
haha.. copied from ipyvue, maybe it's also good to add a link to that in a comment?